airgun/views/oscappolicy.py

```diff
@@ -206,4 +272,4 @@ class HostBreakdownChart(View):
         """Refer to information from the middle of the chart in Oscap Policy
         Details View
         """
-        hosts_breakdown = Text('//div[@id="overview"]/span[@id="pieLabel3"]')
+        hosts_breakdown = Text("//div[contains(@id, 'policy-breakdown-chart')]")
```

Comment:
```suggestion
        hosts_breakdown = Text("//div[@id='policy-breakdown-chart']")
```
I think this will suffice.

Reply:
yeah it does.